packages/hw-wallets/src/trezor/ethereum/index.ts

```
@@ -12,17 +12,21 @@ import {
 
 class TrezorEthereum implements HWWalletProvider {
   network: NetworkNames;
   TrezorConnect: TrezorConnect;
   HDNodes: Record<string, HDKey>;
+  isExtension: boolean;
 
   constructor(network: NetworkNames) {
     this.network = network;
     this.HDNodes = {};
+    this.isExtension = !!(
+      chrome && chrome.runtime && chrome.runtime.getPlatformInfo
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)